README.md

```diff
@@ -151,6 +163,11 @@ This means that the built ROM isn't the same as the base one, so something went
 
 Both of these have the disadvantage that the ordering of the terminal output is scrambled, so for debugging it is best to stick to one thread (i.e. not pass `-j` or `-jN`).
 
+## Changing build options
+
+The project Makefile is fairly configurable and can be used to build other versions of the game or prepare the repo for modding.
+See the Makefile for more information.
```

Comment:
```suggestion
See the options outlined at the top of the Makefile for more information.
```
Might be nice to be very specific here since I get the impression that a lot of people open the makefile, scroll past the options, see some esoteric makefile stuff and then not go back